index.html

```diff
@@ -30,7 +30,7 @@
 
         <li><a href="https://drive.google.com/open?id=0B8noD1hQ2YfZR0E5dGN6RUtlM28" target="_blank">(2016) "Justified Group Belief in Science"</a></li>
        
-        <p><i>Social Epistemology Review & Reply Collective</i> 5(9): 6-12</p>
+        <p><i>Social Epistemology Review &amp; Reply Collective</i> 5(9): 6-12</p>
```

Comment:
If anyone questions why this is absolutely necessary, it's because [the prophets of StackOverflow declared it in their wisdom.](http://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp)